pip/rules.bzl

```diff
@@ -163,9 +171,8 @@ def _deploy_pip_impl(ctx):
         is_executable = True,
         substitutions = {
             "{package_file}": ctx.attr.target[PyDeploymentInfo].package.short_path,
+            "{wheel_file}": ctx.attr.target[PyDeploymentInfo].wheel.short_path,
             "{version_file}": ctx.attr.target[PyDeploymentInfo].version_file.short_path,
-            "{snapshot}": ctx.attr.snapshot,
-            "{release}": ctx.attr.release,
```

Comment:
i am a little concerned about removing functionality.  what did these do, and how are they now accounted for?